backend/graphql/resolvers/testResolvers.ts

```diff
@@ -131,6 +131,25 @@ const testResolvers = {
     ): Promise<string> => {
       return testService.deleteTest(id);
     },
+
+    publishTest: async (
+      _req: undefined,
+      { test_id }: { test_id: string },
```

Comment:
can you use camelCase for consistency?
```suggestion
      { testId }: { testId: string },
```